Automattic/jetpack
# Sync: add Action Scheduler to the list of blacklisted post types

Fixes #12714

#### Changes proposed in this Pull Request:

* Action Scheduler is used to schedule and run background jobs on WordPress, and is used extensively by WooCommerce and WooCommerce extensions.

The data is currently stored in a custom post type, and as such it appears in WordPress.com' activity log even though it does not provide a lot of value to site owners:

![image](https://user-images.githubusercontent.com/426388/60012848-425d4d00-967d-11e9-9986-65f946154d94.png)

This PR stops this post type from being synchronized with WordPress.com, so the events should then stop appearing in the activity log.

**Edit**

I've now removed the additonal blacklisting of the Action Scheduler comments, as those will not be synced once #13079 gets merged.

#### Testing instructions:

* Switch your Woo site to run that branch for a few days.
* You should spot no events from `ActionScheduler` in your activity log.
* You should still be able to use things like the Woo mobile app, or the Woo Store in Calypso.

#### Proposed changelog entry for your changes:

* Sync: add Action Scheduler to the list of blacklisted post types
